ansible/roles/schulcloud-server-init/templates/management-svc.yml.j2

```diff
@@ -5,9 +5,6 @@ metadata:
   namespace: {{ NAMESPACE }}
   labels:
     app: management-deployment
-  annotations:
-    prometheus.io/scrape: 'true'
-    prometheus.io/port: '3333'
```

Comment:
It's fine, but please make you an follow up ticket where you add the new metric endpoint to this service.